### PR TITLE
Updating to RTM builds of xunit and Moq

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @dougbu

File: test/Microsoft.AspNetCore.Identity.EntityFrameworkCore.InMemory.Test/project.json [hunk 1]
```diff
@@ -23,24 +24,11 @@
...
         },
-        "moq.netcore": "4.4.0-beta8",
-        "System.Diagnostics.Process": "4.1.0-*",
         "System.Diagnostics.TraceSource": "4.0.0-*"
       },
-      "imports": [
-        "dnxcore50",
-        "portable-net451+win8"
-      ]
+      "imports": "dotnet5.4"
```
Comment (dougbu) [inline test/Microsoft.AspNetCore.Identity.EntityFrameworkCore.InMemory.Test/project.json]: What's this for?

File: test/Microsoft.AspNetCore.Identity.Test/project.json [hunk 2]
```diff
@@ -23,23 +24,14 @@
           "version": "1.0.0-*",
           "type": "platform"
         },
-        "moq.netcore": "4.4.0-beta8",
         "System.Diagnostics.TraceSource": "4.0.0-*"
       },
       "imports": [
         "dnxcore50",
         "portable-net451+win8"
```
Comment (dougbu) [inline test/Microsoft.AspNetCore.Identity.Test/project.json]: Can these just be `dotnet5.4` too?

**Comment by dougbu** [comment]
:watch:

File: test/Microsoft.AspNetCore.Identity.EntityFrameworkCore.InMemory.Test/project.json (hunk 1 above)
Comment (pranavkm) [inline test/Microsoft.AspNetCore.Identity.EntityFrameworkCore.InMemory.Test/project.json]: The Moq package is referencing a bad build of Castle.Core that supports dotnet5.4. We can do a scan of our imports at a later point when they update their dependency.

**Comment by pranavkm** [comment]
:up: :date:

**Comment by dougbu** [comment]
:shipit:

**Pull request merged**